fireamp.json

```diff
@@ -902,14 +959,4150 @@
+                    "data_path": "action_result.data.*.metadata.links.next",
+                    "data_type": "string",
+                    "example_values": [
+                        "https://api.amp.sourcefire.com/v1/events?limit=10&connector_guid=71841cfb-1892-46b5-b7d7-b18d068246b4&offset=10"
+                    ],
+                    "contains": [
+                        "url"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.metadata.links.self",
+                    "data_type": "string",
+                    "contains": [
+                        "url"
+                    ],
+                    "example_values": [
+                        "https://api.amp.sourcefire.com/v1/events?limit=10&connector_guid=71841cfb-1892-46b5-b7d7-b18d068246b4"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.metadata.results.current_item_count",
+                    "data_type": "numeric",
+                    "example_values": [
+                        10
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.metadata.results.index",
+                    "data_type": "numeric",
+                    "example_values": [
+                        0
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.metadata.results.items_per_page",
+                    "data_type": "numeric",
+                    "example_values": [
+                        10
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.metadata.results.total",
+                    "data_type": "numeric",
+                    "example_values": [
+                        14595
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.version",
+                    "data_type": "string",
+                    "example_values": [
+                        "v1.2.0"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.file_found",
+                    "data_type": "boolean"
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string",
+                    "example_values": [
+                        "File found"
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
             ],
             "versions": "EQ(*)"
         }
-    ]
+    ],
+    "pip_dependencies": {
+        "wheel": [
```

Comment:
Could you confirm if the app can still be installed properly with these wheel changes?

Reply:
actually just saw the integration build. The install looks fine there.